ibm_mq/tests/common.py

```diff
@@ -39,7 +39,7 @@
     'channels': [CHANNEL, BAD_CHANNEL],
 }
 
-INSTANCE_PATTERN = {
+INSTANCE_REGEX_PATTERN = {
```

Comment:
Is it a regex?

Reply:
My bad, mixed the variable naming, should be ok now.